test/runtests.jl

```diff
@@ -58,4 +59,8 @@ include("test_utils.jl")
     include("LaplaceApproximationModule.jl")
     println(" ")
     @info "Ran laplace tests"
+
+    include("sva_optimization.jl")
+    println(" ")
+    @info "Ran optimization test"
```

Comment:
If this is specific to the SparseVariationalApproximation, could you move it into the corresponding file (test/SparseVariationalApproximationModule.jl)?